packages/widgets/src/Transfer/SourceOptions.js

```diff
@@ -1,23 +1,39 @@
+import { spacers } from '@dhis2/ui-constants'
 import React from 'react'
 import propTypes from '@dhis2/prop-types'
 
-import { spacers } from '@dhis2/ui-constants'
+import { EndIntersectionDetector } from './EndIntersectionDetector.js'
 
 export const SourceOptions = ({
     children,
     dataTest,
     sourceEmptyPlaceholder,
```

Comment:
For the `PickedOptions` this seems to be called `selectedEmptyComponent`, here it's `sourceEmptyPlaceholder`. I would expect them to have similar types of names.

Reply:
Yes, I've listed some of the inconsistencies in the PR's description that we need to talk about and whether we want to introduce a breaking change

Reply:
Ah sorry, I missed that. 👍